Change the type in the docs from address to string

cducrest opened the pull request:
and uint256 to number.

berndbohmeier commented on docs/RelayAPI.md:
```diff
@@ -795,13 +795,13 @@ The MetaTransaction object must have the following fields:
-| signature  | bytes   | 65 bytes containing concatenated. v,r,s of the signature           |
+| from       | string   | address of identity contract                                       |
+| to         | string   | the address on which the call of the meta transaction is happening |
+| value      | number   | the amount of wei to be sent along from 'from' to 'to'             |
```
From the schema this is a string.

berndbohmeier replied:
This is because there are only floats in javascript and big numbers can thus not be represented exactly in a number.

berndbohmeier commented on docs/RelayAPI.md:
```diff
@@ -795,13 +795,13 @@ The MetaTransaction object must have the following fields:
+| to         | string   | the address on which the call of the meta transaction is happening |
+| value      | number   | the amount of wei to be sent along from 'from' to 'to'             |
+| data       | string   | the data object encoding the function call including arguments     |
+| nonce      | number   | nonce used for replay protection                                   |
```
Also a string.

berndbohmeier commented:
I am also fine with explaining it better. At the moment I think that address is not helping at all, because there is no type address in json. You need to provide the address as a string.

cducrest commented:
I updated the doc to add a column `JSON Type` describing the basic type in the JSON sent, and specifying that the addresses should be prefixed with `"0x"`. I kept the type column for the "type" we have in python.
I modified the doc where I was responsible for it and I felt there was most confusion. I did not modify the whole doc because it takes time and is not part of the sprint.
